raffo.cpp

```diff
@@ -250,9 +252,9 @@ void RaffoSynth::run(uint32_t sample_count) {
       samples_done = to;
     }
 
-    if (ev) {
-      if (ev->type == midi_type)
-        static_cast<RaffoSynth*>(this)->handle_midi(ev->size, event_data);
+    if (iter) {
+      //if (type == midi_type)
```

Comment:
remove this line too and fix indentation of line below

Reply:
Done!